composer/callbacks/global_straggler_detector.py

```diff
@@ -0,0 +1,873 @@
+from megatron.core.utils import *
```

Comment:
Do we need to do this vs. copying over fns? I don't really like adding megatron as a dep